projects/plugins/social/src/js/editor.js

```diff
@@ -59,6 +71,24 @@ const JetpackSocialSidebar = () => {
 			} }
 		/>
 	);
+
+	// Handle when the review request is dismissed
+	const handleReviewDismiss = useCallback( () => {
+		// Save that the user has dismissed this by calling to the social plugin API method
+		apiFetch( {
+			path: reviewRequestDismissUpdatePath,
+			method: 'POST',
+			data: { dismissed: true },
+		} )
+			.then( () => {
+				// there's nothing to do here.
+			} )
+			.catch( () => {
+				// there's nothing to do here.
+			} );
```

Comment:
They are not handled elsewhere at the moment.

I was not sure if an error is really useful to present to the user because they won't be able to take any action to resolve it 🤔 
I think we could record it for our own purposes though to see if there are errors with the functionality.